### DISCUSSION TITLE
Add a way to get the Attack Damage Attribute of an itemtype before / without a damage event

### DISCUSSION THREAD

**Discussion by Zin-i** [SkriptLang/Skript#6966] [Suggestions] [answered]
My idea is that there should be away to grab the damage that an item / tool would do without having to use the damage event, I've looked through docs and couldn't find anything, & even asked in the SkUnity discord and I was told I'd have to use Reflect to do what I wanted to do. for example, it could be something like:

`set {_damage} to attack damage of {_item}`

Sorry if this is already possible without reflect, but I couldn't really find anything.

**Comment by sovdeeth** [comment, marked as answer]
The issue is this varies significantly depending on the user's potion effects, attributes, and current attack cooldown, as well as whether it's a crit, what armor the opponent is wearing and their attributes/potion effects.

If you just want the attack damage attribute of the weapon, then there's an in-progress PR for that: #5301

**Reply by Zin-i** [reply]
ah thank you! didn't even notice that PR, that's pretty much what I'm going for, my search mustn't have been specific enough.
